applications/common/backend/src/test/java/pl/ds/kyanite/common/components/models/table/TableCellComponentTest.java

```diff
@@ -44,20 +43,20 @@ public void init() {
 
   @Test
   void defaultTableCellComponentModelTest() {
-    TableCellComponent model = context.resourceResolver().getResource(PATH + "/default")
+    TableCellComponent model = context.resourceResolver().getResource(PATH + "/tablebody/tablerow1/default")
```

Comment:
I've made some improvements to the json files, so they actually refer to a full table (not just a standalone cell - this is optional for now, but mandatory for next PR)